### ISSUE TITLE
Only first-level subprojects are imporeted

### ISSUE THREAD

**Issue by strawik** [de-jcup/egradle#288]
I am unable to import multilevel subproject, only first level is imported

e.g. simple structure:
```
root/
|-- server/
|   |-- sever-api
|   |-- server-impl
|-- client/
|   |-- client-api
|   |-- client-impl
|-- build.gradle
|-- settings.gradle
```
Only server and client projects created/imported (just folders, no plugin applied even 'eclipse' ), but missing all java sub-projects (sever-api, server-impl, client-api, client-impl)

**Comment by de-jcup** [comment]
This currently not possible - see https://github.com/de-jcup/egradle/issues/226

As a workaround you can do "Import by standard eclipse way" as described at
https://rawgit.com/de-jcup/egradle/master/egradle-plugin-main/html/user_guide/import.html#import_by_default

If its urgent for you, I will see what I can do.

**Comment by de-jcup** [comment]
@strawik 
I have got some questions:
- Is "server" an eclipse project or just a folder? 
- Which view do you normally use? Package explorer or do you use the Project explorer?
- Do you use working sets?
- It would be nice if you could attach a zip containing an example structure (with gradle files etc)

**Comment by strawik** [comment]
Hello, thanks for workaround. Using standard "gradle eclipse" and importing projects is the way I am doing it now. I am working on very large project, containing Java, C, assembling ... . There is always any issue with buildship, so I wanted to try something new over cli-generated projects.

To your questions:

- server/client are just folders (sometimes with build.gradle with subproject{} section), in maven way, it is pom-packaging, sometimes imported to IDE as simple project

- I am using Navigator, rarely Project Explorer


- I am using Working Sets very often, most of working sets are based on first folder (Server, Client)

- example project attached [GradleTest.tar.gz](https://github.com/de-jcup/egradle/files/1266041/GradleTest.tar.gz)

With the example above, I have 3 working sets

- All projects (containing root, server-api, server-impl, client-api, client-impl), using root as plain resource project to allow acces to files in server and client , as they are virtual sub roots)

- Server (server-api, server-impl)

- Client (client-api, client-impl)

**Comment by de-jcup** [comment]
I think this is now fixed with #425
